test/pdf-view-test.el

```diff
@@ -17,3 +17,12 @@
           (should (numberp (pdf-info-number-of-pages temp)))))
     (when (file-exists-p temp)
       (delete-file temp))))
+
+(ert-deftest pdf-view-cua-copy-region ()
+  (pdf-test-with-test-pdf
+   (pdf-view-mark-whole-page)
+   (should (string-match "PDF Tools\\(?:.\\|\n\\)*in memory"
+			 (let (kill-ring)
```

Comment:
`string-match-p`

Seems like it should succeed.

Reply:
I forgot:  `pdf-view-mark-whole-page` tries to redisplay the image, which does not work in a batch'd Emacs.